### ISSUE TITLE
Play next Video automatically (like Youtube-Website)

### ISSUE THREAD

**Issue by h0d3nt3uf3l** [Kolifanes/plugin.video.youtube#34]
[@Kodiforum:SteveJobs] Is there any way, when a video has ended, to play the next one automatically? (just like in the original YouTube website)
-- Marked for next release from h0d3nt3uf3l

**Comment by andreskru** [comment]
I like this :)

**Comment by jimporter** [comment]
You can open the context menu in Kodi and select "Play from here", can't you? That should play everything in the list starting at the item you selected.

**Comment by h0d3nt3uf3l** [comment]
@jimporter you're right, but the feature what is requested here is like in youtube the automatic mix if you play a video, which is played after the first video is played. I use this feature very often and it seems like other users too :)

**Comment by nelsonec87** [comment]
+1 for automatic mix
